csharp-basics/exercises/ClassesAndObjects/Dog/Dog.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+namespace Dog
+{
+	public class Dog
+	{
+		public string Name { get; set; }
+		public string? Sex { get; set; }
+		public string? Mother { get; set; }
+		public string? Father { get; set; }
+
+		public Dog(string name, string? sex, string? mother, string? father)
```

Comment:
Tev būtu jābūt 2 dažādiem konstruktoriem - Change the Dog class so that each dog has a mother and a father. Create a class Dog. Dogs should have a name, and a sex.